web/pdf_print_service.js

```diff
@@ -200,7 +211,17 @@
       if (!activeService) {
         console.error('Expected print service to be initialized.');
       }
-      activeService.renderPages().then(startPrint, abort);
+      var activeServiceOnEntry = activeService;
+      activeService.renderPages().then(function() {
+        activeServiceOnEntry.throwIfInactive();
+        return startPrint(activeServiceOnEntry);
+      }).catch(function() {
+        // Ignore any error messages.
+      }).then(function() {
+        if (activeServiceOnEntry.active) {
+          abort();
```

Comment:
Ignore the above. Is `startPrint` actually `performPrint` -- I read startPrint as something that initiating print?